website/content/docs/platform/k8s/csi/index.mdx

```diff
@@ -69,6 +69,8 @@ long-lived default service account token, with an issuer different to the tokens
 Vault CSI Provider will create. But one Kubernetes auth mount can only be
 configured to validate a single issuer value.
 
+-> **Deprecated:** The issuer has been deprecated and will be removed in a future release.
+
```

Comment:
I think I've usually seen a deprecation notice like this at the top of a section instead of the bottom. I'd also suggest that we mention the vault version, something along the lines of "The issuer parameter has been deprecated as of Vault 1.9 and will be....".

Reply:
> See my previous two comments about the deprecated notice and section placement. Also I think "[Discovering the service account issuer](https://github.com/hashicorp/vault/blame/main/website/content/docs/auth/kubernetes.mdx#L104)" still needs to be updated here.

Can this section be removed entirely?

Reply:
I'd say since these parameters haven't been removed yet we should keep the "Discovering the service account issuer" section.